cmd/app/main.go

```diff
@@ -76,6 +77,17 @@ func main() {
 		log.Panicf("failed to register gateway endpoint: %v", err)
 	}
 
+	if err := d.MUX.HandlePath(http.MethodGet, "/", func(w http.ResponseWriter, r *http.Request, _ map[string]string) {
```

Comment:
Are we sure that this won't hide the other handlers?

Reply:
no, i will try to test that,
i was unsure if that is the correct way to do that